odl_video/settings.py

```diff
@@ -570,5 +570,10 @@ def get_all_config_keys():
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+SECURE_CROSS_ORIGIN_OPENER_POLICY = get_string(
+    "SECURE_CROSS_ORIGIN_OPENER_POLICY",
+    "same-origin",
```

Comment:
We should default this to the value that we want, which is "same-origin-allow-popups", otherwise we risk regressing again.